### PR TITLE
Add missing webspace parameter for CacheClearToolbarAction in the SnippetAreas view

### LINKED ISSUE #6041: Blank screen on default snippets overview
| Q | A
| --- | ---
| Sulu Version | 2.3.0-RC1
| PHP Version | 7.4.x
| DB Version | MySQL 5.7.x
| Browser Version | Chrome, latest version

# Actual Behavior
Blank screen when going to default snippets overview. Console error logged:
```
react-dom.production.min.js:216 TypeError: Cannot read property 'webspaceKey' of undefined
     at $le.getNode (CacheClearToolbarAction.js:31)
     at i.render (SnippetAreas.js:160)
     at Te (mobx.module.js:1015)
     at index.module.js:847
     at me (mobx.module.js:801)
     at e.track (mobx.module.js:1833)
     at i.n [as render] (index.module.js:841)
     at Hs (react-dom.production.min.js:187)
     at Fs (react-dom.production.min.js:186)
     at Ha (react-dom.production.min.js:269)
```

# Expected Behavior
Seeing the default snippets overview

# Steps to Reproduce
Go to pages -> default snippets

### PR DESCRIPTION
**Pull request by Prokyonn**
| Q | A
| --- | ---
| Bug fix? | yes
| New feature? | no
| BC breaks? | no
| Fixed tickets | fixes https://github.com/sulu/sulu/issues/6041
| License | MIT

#### What's in this PR?

Adds the missing parameter `webspaceKey` in the `CacheClearToolbarAction` for the `SnippetAreas` view.

### CODE REVIEW

File: src/Sulu/Bundle/SnippetBundle/Resources/js/views/SnippetAreas/SnippetAreas.js [hunk 1]
```diff
@@ -32,7 +32,9 @@ class SnippetAreas extends React.Component<ViewProps> {
         }
 
         this.snippetAreaStore = new SnippetAreaStore(webspace);
-        this.cacheClearToolbarAction = new CacheClearToolbarAction();
+        this.cacheClearToolbarAction = new CacheClearToolbarAction({
+            webspaceKey: webspace,
```
Comment (alexander-schranz) [inline src/Sulu/Bundle/SnippetBundle/Resources/js/views/SnippetAreas/SnippetAreas.js]: I think this should still be optional in cases where we don't have a webspace key.

File: src/Sulu/Bundle/LocationBundle/Resources/js/package.json [hunk 2]
```diff
@@ -6,7 +6,9 @@
     "dependencies": {
         "classnames": "^2.2.5",
         "leaflet": "^1.5.1",
-        "react-leaflet": "^3.1.0"
+        "react-leaflet": ">=3.1.0 <3.2.0 || ^3.2.1",
```
Comment (niklasnatter) [inline src/Sulu/Bundle/LocationBundle/Resources/js/package.json:9]: this is a workaround to fix the build because of https://github.com/PaulLeCam/react-leaflet/issues/879. we should revert the change when https://github.com/PaulLeCam/react-leaflet/issues/879 is fixed.

**Review by alexander-schranz — APPROVED** [review]
